app/models/job.ts

```diff
@@ -1041,6 +1041,9 @@ export class Job extends DBRecord implements JobRecord {
       numInputGranules: this.numInputGranules,
       jobID: this.jobID,
     };
+    // need this line to prevent null values fro showing up in data expiration field
+    Object.keys(serializedJob).forEach((k) => serializedJob[k] == null && delete serializedJob[k]);
```

Comment:
There is a helper function `removeEmptyProperties` that you could use.

Reply:
Done.